ACC Algorithm/ACCAlgorithMain.c

```diff
@@ -0,0 +1,154 @@
+#include<stdio.h>
+
+//Globals
+float hostVel;
+float hostVelPrev; //Host velocity in the previous time step
+float integral = 0; //Needs to be global as it is used in the calculation of next step
+float errorPrev = -999.0; //Previous value of PID error
+float leadDistPrev = 0;
+float overshoot = 0;
+float leadTime = 0;
+
+//Calibratables
+float timeStep = 0.1; //Time step of system
+float simTime = 20.0; //Simulation time in seconds
```

Comment:
may move these to a different header file... maybe `calibrations.h` or smth...

Reply:
Made a header file with function headers, moved enums, structs, and consts